src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -174,6 +175,7 @@ export const SueReportLocation = ({
           name="houseNumber"
           label={texts.sue.report.houseNumber}
           placeholder={texts.sue.report.houseNumber}
+          textContentType="off"
```

Comment:
why was it necessary to set this to `off`?

Reply:
Because here we do not get a correct suggestion from the iOS system. Therefore, I have closed the suggestion.